src/components/Account/OfferList.tsx

```diff
@@ -47,45 +54,60 @@ interface OfferListItemProps {
 
 function OfferListItem(props: OfferListItemProps) {
   const [hovering, setHoveringStatus] = React.useState(false)
+  const isSmallScreen = useIsMobile()
+  const isTinyScreen = useIsSmallMobile()
+
   return (
     <ListItem
       onMouseEnter={() => setHoveringStatus(true)}
       onMouseLeave={() => setHoveringStatus(false)}
       style={props.style}
     >
-      <ListItemIcon>
-        <BarChartIcon />
-      </ListItemIcon>
-      <ListItemText
-        primary={
-          <span style={{ fontWeight: "bold" }}>
-            Sell&nbsp;&nbsp;
-            <SingleBalance assetCode={props.offer.selling.code} balance={props.offer.amount} inline />
-            &nbsp;&nbsp;for&nbsp;&nbsp;
-            <SingleBalance
-              assetCode={props.offer.buying.code}
-              balance={String(BigNumber(props.offer.amount).mul(props.offer.price))}
+      <HorizontalLayout alignItems="center" grow={1} wrap={"wrap"}>
```

Comment:
Phew, that's quite a bit of extra elements just to make it look nicer on tiny screens... Let's have a look if we can simplify that 😉